### PR TITLE
docs: FAQs added

### PR DESCRIPTION
**Pull request by aravindavk**
Signed-off-by: Aravinda Vishwanathapura <aravinda@kadalu.io>

### CODE REVIEW

File: docs/faqs.adoc [hunk 1]
```diff
@@ -0,0 +1,43 @@
+= Kadalu Storage FAQs
+
+== Is Kadalu Storage the same as GlusterFS except for the management layer?
+
+Kadalu Storage is an opinionated Storage solution. It uses only the core filesystem from GlusterFS. Kadalu Storage brings good parts of GlusterFS with added enhancements like improved Quota, Management layer and others.
```
Comment (sac) [inline docs/faqs.adoc]: among others*

File: docs/faqs.adoc [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+Along with the additional features, the default configurations/values will differ. All the enhancements available with Kadalu Storage are also Open Source.
```
Comment (sac) [inline docs/faqs.adoc:7]: Can we add a line saying there will be more intuitive CLI and syntax.

**Review by sac — APPROVED** [review]

File: docs/faqs.adoc (hunk 2 above)
Comment (aravindavk) [reply docs/faqs.adoc:7]: Done

File: docs/faqs.adoc (hunk 1 above)
Comment (aravindavk) [reply docs/faqs.adoc]: Fixed.

**Pull request merged**